oemoflex/postprocessing.py

```diff
@@ -270,16 +270,7 @@ def get_sequences_by_tech(results):
     sequences_by_tech = {}
 
     # Get a list of internal busses for all 'ReservoirWithPump' nodes to be ignored later
-    to_nodes = []
-    for k, s in sequences.items():
-        to_nodes.append(k[1])
-
-    internal_busses = []
-    for node in to_nodes:
-        if hasattr(node, 'subnodes'):
-            # Only get the subnodes of type Bus
-            internal_bus = [n for n in node.subnodes if isinstance(n, Bus)]
-            internal_busses.extend(internal_bus)
+    internal_busses = get_busses_in_subnodes(sequences)
```

Comment:
'sequences' has already a special structure here, see above. So, extracting the list building results in a probably non-reusable function!

Reply:
Maybe move to a sub function because only usable in this context.